util/llpcPipelineDumper.cpp

```diff
@@ -905,24 +905,60 @@ void PipelineDumper::DumpGraphicsPipelineInfo(
 }
 
 // =====================================================================================================================
-// Builds hash code from graphics pipline build info.
+// Builds hash code from graphics pipline build info.  If stage is a specific stage of the graphics pipeline, then only
+// the portions of the pipeline build info that affect that stage will be included in the hash.  Otherwise, stage must
+// be ShaderStageInvalid, and all values in the build info will be included.
 MetroHash::Hash PipelineDumper::GenerateHashForGraphicsPipeline(
     const GraphicsPipelineBuildInfo* pPipeline,   // [in] Info to build a graphics pipeline
-    bool                            isCacheHash   // TRUE if the hash is used by shader cache
+    bool                            isCacheHash,  // TRUE if the hash is used by shader cache
+    uint32_t                        stage         // [in] The stage for which we are building the hash.
```

Comment:
uint32_t stage
)
->
uint32_t stage)
No "const" qualifier, so we use this style.